cosmos/config.py

```diff
@@ -70,6 +71,8 @@ def __post_init__(self, dbt_project_path: str | Path | None) -> None:
                 DeprecationWarning,
             )
         self.project_path = Path(dbt_project_path) if dbt_project_path else None
+        # allows us to initiate this attribute from Path objects and str
+        self.dbt_ls_path = Path(self.dbt_ls_path) if self.dbt_ls_path else None
```

Comment:
@flinz, could we add a unit test to cover the support for `dbt_ls_path` to be a string so we don't revert this behaviour unintentionally in future?

Reply:
@tatiana I added one simple test that invokes the constructor with a string argument.
https://github.com/flinz/astronomer-cosmos/commit/c69e0c7d6f65b38c20e87e11aa0592657d68fe8c